src/main/java/io/github/aquerr/steamwebapiclient/response/PlayerGetRecentlyPlayedGamesResponse.java

```diff
@@ -0,0 +1,44 @@
+package io.github.aquerr.steamwebapiclient.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Class representing the result of <a href="https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames">https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames</a>
+ */
+@Data
+public class PlayerGetRecentlyPlayedGamesResponse implements SteamWebApiResponse {
```

Comment:
I would recommend to change it to "PlayerRecentlyPlayedGamesResponse"

Reply:
fixed